accelerator-core/src/main/java/com/tokbox/android/otsdkwrapper/wrapper/OTWrapper.java

```diff
@@ -705,19 +706,18 @@ public void setPublishingFPS(int FPS) {
         mPreviewConfig = new PreviewConfig.PreviewConfigBuilder().framerate(frameRate).build();
       }
       newPreview = mPreviewConfig;
-      boolean isPublishing = this.isPublishing;
-      boolean isPreviewing = this.isPreviewing;
-      if (isPublishing) {
+
+      if (this.isPublishing) {
```

Comment:
I think that this change will introduce a bug. It looks like these local variables where added on purpose...
```
boolean isPublishing = this.isPublishing;	
boolean isPreviewing = this.isPreviewing;
```
...because `startPreview/stopPreview` and `startPublishingMedia/stopPublishingMedia` are changing `isPublishing/isPreviewing` class field value and this method should react to old value (value of the time method being called, not valued modified by `startPreview/stopPreview/startPublishingMedia/stopPublishingMedia` while this method is running)

Reply:
these were conflicting with outer scope variables with same name.

Reply:
i renamed the variables to avoid variable shadowing.

Reply:
I think this was done on purpose to make sure that subsequent method calls (done inside `setPublishingFPS` method) will not break the flow. 
For me it looks like a potencial bug, so please just make sure you test this logic.